lib/db/schema/patch-022-023.sql

```diff
@@ -0,0 +1,140 @@
+-- This migration:
+--
+-- * Changes the type of the devices.callbackPublicKey column
+-- * Add a new devices.callbackAuthKey column
+-- * Updates the accountDevices stored procedure with the new column
+-- * Alters three stored procedures, createDevice, updateDevice and
+--   sessionWithDevice to work with the new columns
+
+-- 2-steps callbackPublicKey change: we avoid junk values, and we don't block
+-- concurent queries.
+
+ALTER TABLE devices
+CHANGE callbackPublicKey oldCallbackPublicKey BINARY(32),
+ADD COLUMN callbackPublicKey VARCHAR(88),
+ADD COLUMN callbackAuthKey VARCHAR(24);
```

Comment:
nit: if we expect the values to be always exactly the given lengths, we could consider using a `CHAR` column here to save one byte per row.